sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,460 @@
+-- Query for firefox_ios_derived.feature_usage_metrics_v1
+-- For more information on writing queries see:
+-- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE submission_date DATE DEFAULT "2023-12-07";
+
+
+WITH metrics_ping_distinct_client_count AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS metrics_ping_client_count
```

Comment:
```suggestion
    COUNT(DISTINCT client_info.client_id) AS metrics_ping_distinct_client_count
```
I'd maybe make it explicit in the field name that this is a distinct count.